hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml

```diff
@@ -14,7 +14,7 @@ market: null
 # What leverage to used
 leverage: null
 
-# Position mode to use, one-way or hedge position mode
+# Position mode to use, one-way or hedge position mode (If disabled, one-way position mode would be used. (True or False))
```

Comment:
This is unclear on what values are expected for this parameter. Is it one-way/hedge or True/False ?

Reply:
I update the additional info.